packages/dapp/src/apis/civilTCR.ts

```diff
@@ -53,6 +53,9 @@ export async function approve(
   const token = await tcr.getToken();
   const amountBN = ensureWeb3BigNumber(amount);
   console.log("token address: " + token.address);
+  if ((await token.getBalance()) < amountBN) {
```

Comment:
probably worth adding the CVL -> multisig transfer into the series of apply transactions. not a big deal though.

Reply:
Agreed, or we should at least offer to do it - and then check that you have enough CVL. But potentially the person applying might not be the person with CVL to send to multisig, so might have to happen separately. We can work out that flow later.